rust/app/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "vmss-prototype"
+version = "0.1.0"
+authors = ["Brook Tamir <t-brooktamir@microsoft.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+command = { path = "../command" }
+# runner = { path = "../runner" }

```

Comment:
remove commented fragments from all .toml files

It's rare that you need to leave leave some code as commented.  In such case you must almost always have
comments describing why the code is commented and usually associate with some open workitem